app/models/update_certificate_notification.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateCertificateNotification < AggregatedEvent
```

Comment:
This should be an event changing a specific field to true. So the event should be named more specifically, something along the lines of `CertificateNotificationSentEvent` ... it also makes it easier (for whoever is reading the code) to associate the field on the object with the corresponding event